draft-ietf-acme-acme.md

```diff
@@ -2046,18 +2046,18 @@ need to specify which types of identifier they apply to.
 
 ## Key Authorizations
 
-Several of the challenges in this document make use of a key authorization
+All challenges defined in this document make use of a key authorization
 string.  A key authorization is a string that expresses a domain holder's
 authorization for a specified key to satisfy a specified challenge, by
 concatenating the token for the challenge with a key fingerprint, separated by a
 "." character:
 
 ~~~~~~~~~~
-key-authz = token || '.' || base64url(JWK_Thumbprint(accountKey))
+  key-authorization = token || '.' || base64url(JWK_Thumbprint(accountKey))
```

Comment:
This should be `keyAuthorization`.

Reply:
Thanks for the feedback @shred. I understood this line as an abstract definition of a term. Actually, it was never identical to the object key before. But it is the value for `keyAuthorization`. Maybe it would be helpful to use the same name?

Reply:
You are right, it is abstract and was never identical. But I think it would enhance readability, and there is also a camel-cased `accountKey` used in the same expression.

Reply:
I agree with @shred - I think we should be consistent with `accountKey` and use `keyAuthorization` in the abstract definition.